src/traces/carpet/map_2d_array.js

```diff
@@ -8,6 +8,8 @@
 
 'use strict';
 
+var isArrayOrTypedArray = require('../../lib').isArrayOrTypedArray;
```

Comment:
this file is `map_2d_array` - and we're not supporting typed 2D arrays, at least not yet, right?

Reply:
Good eyes. Only the inner arrays should be allowed to be typed arrays.